cmd/kube-apiserver/app/options/options.go

```diff
@@ -94,7 +94,7 @@ func NewServerRunOptions() *ServerRunOptions {
 		EnableLogsHandler:      true,
 		EventTTL:               1 * time.Hour,
 		MasterCount:            1,
-		EndpointReconcilerType: string(reconcilers.MasterCountReconcilerType),
```

Comment:
update the help text on the `apiserver-count` flag to indicate it only applies when using `--endpoint-reconciler-type=master-count`?

do we also want to mark the master-count reconciler deprecated this release, for removal in 1.13? I can't think of a reason we'd want to keep it.

Reply:
I'll put that into this PR

Reply:
I updated the release note as well.